app/assets/stylesheets/darkswarm/_shop-product-rows.css.scss

```diff
@@ -141,37 +83,53 @@
         .summary-header {
           padding-left: 7.9375rem;
 
-          @media all and (max-width: 768px) {
-            padding-left: 4.9375rem;
-          }
+          .product-producer {
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            font-style: italic;
 
-          @media all and (max-width: 640px) {
-            padding-left: 0.9375rem;
-          }
+            a {
+              color: $teal-500;
 
-          small {
-            font-size: 80%;
+              &:hover, &:focus, &:active {
+                color: $teal-600;
+                text-decoration: underline;
+              }
+            }
           }
 
           h3 {
             font-size: 1.5rem;
-            margin: 0;
           }
 
           h3 a {
-            color: #222;
+            color: $orange-500;
 
-            i {
-              @include csstrans;
-
-              font-size: 0.6em;
+            &:hover, &:focus, &:active {
+              color: $orange-600;
+              text-decoration: underline;
             }
+          }
 
-            &:hover, &:focus, &:active {
-              color: $clr-brick;
+          .product-description {
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
```

Comment:
these three properties are duplicated in `.product-producer` so I think it's worth extracting this pattern and giving it a name.

Reply:
I usually start DRYing something when I use it for the third time. These properties are definitely right for the producer. But for the product description, it's only an approximation of the design created by Yuko. She wanted a "more" button at the end and display two lines on mobile. That is far more complex and needs Javascript. I avoided that complexity for now but maybe someone wants to change this logic in the future. And then the extraction would just be unnecessary complexity.

In the words of the following article, I would call this *accidental* duplication.
https://www.justinweiss.com/articles/i-dry-ed-up-my-code-and-now-its-hard-to-work-with-what-happened/